.travis.yml

```diff
@@ -1,6 +1,6 @@
 sudo: required
 
-language: scala
```

Comment:
And of course that was written by me 🤦‍♂️ 

[Reading in to it a bit more](https://docs.travis-ci.com/user/languages/scala/), it seems that if we set the language to `scala` it tries to build using `sbt`. However, in our case we want to build from within Docker, thus `bash` is correct. Sorry about the confusion.